spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'tempfile'
 require 'pry'
 require 'thread'
+require 'pathname'
```

Comment:
It's ok. One of these days we'll get around to fixing all the quote issues. This is what happens when you change your style guide in the middle of a long running project. :)